array/array-test.js

```diff
@@ -0,0 +1,86 @@
+var ObserveArray = require("./array");
+var ObserveObject = require("../object/object");
+var QUnit = require("steal-qunit");
+
+QUnit.module("can-observe/array");
+
+var classSupport = (function() {
+	try {
+		eval('"use strict"; class A{};');
+		return true;
+	} catch (e) {
+		return false;
+	}
+
+})();
+
+if(classSupport) {
+    QUnit.test("observe.Array basics", function(){
+        class TodoList extends ObserveArray {
```

Comment:
This will be a parse error, so putting it conditionally like this doesn't help. Putting the class tests in a separate file and conditionally requiring them should work though.